crabml-core/src/backends/cpu/buf/api.rs

```diff
@@ -7,6 +7,7 @@ use super::buf_f16::f16_buf_from_bytes;
 use super::buf_f16::quantize_f32_f16;
 use super::buf_f32::f32_buf_from_bytes;
 use super::buf_f32::vec_dot_f32_f32;
+use super::QuantBufQ4K;
```

Comment:
this import could be keep same style like the following lines, like `use crate::backends::cpu::buf::QuantBufQ4K;` : )